### PR TITLE
feat: added nprogress, docker image and removed old manager

### PR DESCRIPTION
**Pull request by akhilmhdh**
## Explain the feature/fix

1. Docker image issues
2. Added NProgress
3. Removed old manager codebase

## Does this PR introduce a breaking change

No

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/manager/src/pages/_app.tsx [hunk 1]
```diff
@@ -43,10 +47,30 @@ yup.addMethod(yup.string, 'alphaNumbericOnly', function noWhitespace() {
   });
 });
 
+NProgress.configure({ showSpinner: false });
```
Comment (hybridx) [inline packages/manager/src/pages/_app.tsx:50]: @akhilmhdh you can keep the default spinner it's not going to be a problem according to me

**Review by hybridx — CHANGES_REQUESTED** [review]

File: packages/manager/src/pages/_app.tsx (hunk 1 above)
Comment (hybridx) [reply packages/manager/src/pages/_app.tsx:50]: Rest LGTM

Comment (akhilmhdh) [reply packages/manager/src/pages/_app.tsx:50]: Spinner is removed due to navbar.

**Review by arkaprovob — APPROVED** [review]
This is not the master branch. Let's deploy his changes in the dev environment if something needs to be changed, and we can do that anytime.

**Review by soumyadip007 — APPROVED** [review]

**Pull request merged**